pkg/storage/stores/indexshipper/compactor/compactor.go

```diff
@@ -506,8 +551,13 @@ func (c *Compactor) CompactTable(ctx context.Context, tableName string, applyRet
 		return fmt.Errorf("index processor not found for index type %s", schemaCfg.IndexType)
 	}
 
-	table, err := newTable(ctx, filepath.Join(c.cfg.WorkingDirectory, tableName), c.indexStorageClient, indexCompactor,
-		schemaCfg, c.tableMarker, c.expirationChecker, c.cfg.UploadParallelism)
+	sc, ok := c.containers[schemaCfg.ObjectType]
+	if !ok {
+		return fmt.Errorf("storeContainer not found for %s", schemaCfg.ObjectType)
```

Comment:
Will a user be able to understand what this error means?

Reply:
might not be helpful unless they grok the code. I'll try to add more context to the log message